### DISCUSSION TITLE
fpm-docs in fpm.fortran-lang.org

### DISCUSSION THREAD

**Discussion by aslozada** [fortran-lang/fpm-docs#17] [General]
Should _fpm-docs_ replace the page [packaging](https://fpm.fortran-lang.org/page/Packaging.html) in fpm.fortran-lang.org?

I noticed that the information on the page ``packaging`` in fpm.fortran-lang.org is already contained in _fpm-docs_. On the other hand, in _fpm-docs_ this information is more detailed.

I think that _fpm-docs_ could be **official** in fpm.fortran-lang.org

**Comment by awvwgk** [comment]
It's done, now *fpm-docs* is official.

**Reply by milancurcic** [reply]
You rock--I shared it on Twitter: https://twitter.com/fortranlang/status/1470864149295120395.
